src/Kdyby/Console/DI/ConsoleExtension.php

```diff
@@ -81,7 +81,8 @@ public function loadConfiguration()
 
 		Nette\Utils\Validators::assert($config, 'array');
 		foreach ($config['commands'] as $command) {
-			$def = $builder->addDefinition($this->prefix('command.' . md5(Nette\Utils\Json::encode($command))));
+			$entityName = ($command instanceof Statement) ? $command->getEntity() : '';
+			$def = $builder->addDefinition($this->prefix('command.' . md5($entityName.Nette\Utils\Json::encode($command))));
```

Comment:
I don't even remember why I did it in this way. Maybe this would be better?

```php
foreach ($config['commands'] as $i => $command) {
    $def = $builder->addDefinition($this->prefix('command.' . $i));
```